README.md

```diff
@@ -212,9 +212,9 @@ More info: http://stackoverflow.com/a/35098040/1683164
 6. Install the latest version of boost, specificly the required static libraries
     ```
     cd
-    wget http://sourceforge.net/projects/boost/files/boost/1.63.0/boost_1_63_0.tar.bz2
-    tar xjf boost_1_63_0.tar.bz2
-    cd boost_1_63_0
+    wget http://sourceforge.net/projects/boost/files/boost/1.64.0/boost_1_64_0.tar.bz2
```

Comment:
use http://dl.bintray.com/boostorg/release/1.64.0/source/boost_1_64_0.tar.bz2 instead of sourceforge, as pointed in monero-project/monero#3316